### PR TITLE
Feature/slack system service

### PR DESCRIPTION
**Pull request by benmaidel**
- added rosmaster independent python scripts

### CODE REVIEW

**Comment by fmessmer** [comment]
what's the advantage here?
it's not ros-independent...you use rospy in both scripts...?

please use/test/integrate #5 when testing slack interface...it's a significant improvement against online/offline situations...it might even make the re-write done here obsolete

File: slack_ros/scripts/slack_service.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import sys
+import os
+import threading
+import signal
+import zmq
+
+import rospy
+from std_msgs.msg import String
+from slack_interpreter.utils import unescape
+from slack_interpreter.modules_system import *
```
Comment (fmessmer) [inline slack_ros/scripts/slack_service.py:13]: this dependency is not very nice...now this depends on a package in msh

File: slack_ros/scripts/slack_service.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+    signal.signal(signal.SIGINT, signal_handler)
+
+    slack_srv = SlackService(sys.argv[1], sys.argv[2], sys.argv[3])
+    mediator = InterpreterMediator(slack_srv)
+    
+    while not SIGINT:
+
+        try:
+            rospy.get_master().getPid()
```
Comment (fmessmer) [inline slack_ros/scripts/slack_service.py:166]: this makes it rosdependend again...

**Review by fmessmer — CHANGES_REQUESTED** [review]
also, install tags are missing

**Comment by fmessmer** [comment]
As discussed during lunch:
@ipa-bnm please move your new code to `msh` repo directly...it's a total re-write thus, it does not need to live in the repo we once forked....
This also saves us a few entries in the `.rosinstall` files as well!

File: slack_ros/scripts/slack_service.py (hunk 2 above)
Comment (benmaidel) [reply slack_ros/scripts/slack_service.py:166]: sure, to be more precise, the node should be roscore independent. It's ok to use rospy here

**Comment by fmessmer** [comment]
will pull this feature into msh, thus this PR won't get merged here!

**Pull request closed without merging**